server/filestore.go

```diff
@@ -104,26 +104,27 @@ type psi struct {
 }
 
 type fileStore struct {
-	mu      sync.RWMutex
-	state   StreamState
-	ld      *LostStreamData
-	scb     StorageUpdateHandler
-	ageChk  *time.Timer
-	syncTmr *time.Timer
-	cfg     FileStreamInfo
-	fcfg    FileStoreConfig
-	prf     keyGen
-	aek     cipher.AEAD
-	lmb     *msgBlock
-	blks    []*msgBlock
-	bim     map[uint32]*msgBlock
-	psim    map[string]*psi
-	hh      hash.Hash64
-	qch     chan struct{}
-	cfs     []ConsumerStore
-	sips    int
-	closed  bool
-	fip     bool
+	mu       sync.RWMutex
+	state    StreamState
+	ld       *LostStreamData
+	scb      StorageUpdateHandler
+	ageChk   *time.Timer
+	syncTmr  *time.Timer
+	cfg      FileStreamInfo
+	fcfg     FileStoreConfig
+	prf      keyGen
+	aek      cipher.AEAD
+	lmb      *msgBlock
+	blks     []*msgBlock
+	bim      map[uint32]*msgBlock
+	psim     map[string]*psi
+	hh       hash.Hash64
+	qch      chan struct{}
+	cfs      []ConsumerStore
+	sips     int
+	closed   bool
+	fip      bool
+	received bool
```

Comment:
Let's call it receivedAny or receivedFirst for clarity.

Reply:
agree. changed to `receivedAny`